docs/reference/analysis/tokenizers/pathhierarchy-tokenizer-examples.asciidoc

```diff
@@ -0,0 +1,195 @@
+[[analysis-pathhierarchy-tokenizer-examples]]
+=== Example tokenizing file paths
+
+A common use-case for the `path_hierarchy` tokenizer is filtering results by 
+file paths. If indexing a file path along with the data, the use of the 
+`path_hierarchy` tokenizer to analyze the path allows filtering the results 
+by different parts of the file path string.
+
+
+This example configures an index to have two custom analyzers and applies
+those analyzers to multifields of the `file_path` text field that will 
+store filenames. One of the two analyzers uses reverse tokenization.
+Some sample documents are then indexed to represent some file paths
+for photos inside photo folders of two different users.
+
+
+[source,js]
+--------------------------------------------------
+PUT /file-path-test/
```

Comment:
should be have just `file-path-test`  instead of `/file-path-test/` for consistency? In this and following requests.